_posts/2017-09-14-survey-results.md

```diff
@@ -0,0 +1,218 @@
+![Bar chart, scale of 1 (easier) to 5 (harder): How easy did you find it to integrate the Pa11y Dashboard with your workflow? 50% for 1; 15% for 2 and 3; 25% for 4](asset/image/2017-survey/3-integrate.png)
+
+We asked people what would make it easier for them to set up and use Pa11y Dashboard. Respondents overwhelmingly said they want better documentation. We also asked what would help better integrate Pa11y Dashboard with user workflows. Most respondents had specific features in mind that would help them get information to stakeholders, particularly for larger websites or collections of web properties. 
+
+* email reports
+* Being able to sort elements on the dashboard, and having multiple dashboard "pages". In one page we could see all of our competition, in another our own site pages, if that makes sense
+* display all errors and issues
+* multiple urls support
+
+We asked what people like about Pa11y Dashboard. Respondents find the visual comparisons helpful:
+
+* Graphs
+* great way to overview our sites and see historical reports
+* Ease of use
+* Seeing our errors go down, or stay at zero. Keeping an eye on the competition, being able to have a history of improvements we've made to the site
+* easy to glance at issues, more friendly to non-developers
+* Easy
+
+We also asked what people don’t like so much about Pa11y Dashboard. Most of these respondents would like to see improvements to the UX of the dashboard itself. One user reported stability issues. 
+
+* limited to last 30 days
+* Colours seems to be clashing.
+* If left alone it tends to fail (widgets not load any data)
+* not able to see all errors and issues
+* Limited
+
+
+## Pa11y Webservice
+
+The Pa11y webservice is the JSON-based webservice that supports Pa11y Dashboard. It allows you to integrate Pa11y with existing dashboards, build your own dashboard-like front ends, or repurpose Pa11y report data for consumption by other services. 
+
+Less than a quarter of our respondents have used the Pa11y Webservice. 
+
+![Pie chart: Have you ever used the Pa11y webservice? 17% yes; 83% no](asset/image/2017-survey/4-pa11y-webservice.png)
+
+Reasons for not using the Webservice were varied, but most people (80%) just hadn’t heard of it. Others said that it doesn’t do what they need, or that another tool fits their purposes better. 
+
+The Webservice is supposed to complement DIY solutions to dashboard services, so we weren’t expecting many people to have used this on its own. Pa11y Dashboard uses the Webservice under the hood and so the Dashboard (being a complete solution) is what the majority of users are likely to be interested in. 
+
+Our respondents who have used the Pa11y Webservice report it mostly being easy to use and set up:
+
+![Bar chart, scale of 1 (easier) to 5 (harder): How easy did you find it to set up / use the Pa11y webservice? 70% for 2; 30% for 4](asset/image/2017-survey/4-setup-use.png)
```

Comment:
This chart is a lot bigger than the others and overflows the container element. I think this highlights that we need to set a `max-width` of `100%` on images in the `.news-article` container 😁

Reply:
i forgot to resize the image before committing! i did it in another commit, but I'll update the css too, good thinking 👍